frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion/tests/PublicOrEmbeddedQuestion.common.unit.spec.tsx

```diff
@@ -1,24 +1,40 @@
 import userEvent from "@testing-library/user-event";
 
 import { screen, waitForLoaderToBeRemoved } from "__support__/ui";
+import { useSetEmbedFont } from "metabase/public/hooks";
 
 import { type SetupOpts, setup } from "./setup";
 
+jest.mock("metabase/public/hooks/use-set-embed-font", () => ({
+  useSetEmbedFont: jest.fn(),
+}));
+
 const FAKE_UUID = "123456";
 
 const QUESTION_NAME = "Public question";
 
-function setupCommon(opts?: Partial<SetupOpts>) {
-  return setup({
+async function setupCommon(opts?: Partial<SetupOpts>) {
+  const useSetEmbedFontMock = useSetEmbedFont as jest.Mock;
+
+  await setup({
     ...opts,
     questionName: QUESTION_NAME,
     uuid: FAKE_UUID,
   });
+
+  return { useSetEmbedFontMock };
 }
 
 describe("PublicOrEmbeddedQuestion", () => {
+  it("sets embed font", async () => {
```

Comment:
I'm not a fan of a test like this since it relies on a specific kind of implementation. If we change the way we reads `font` this test could fail, but the Cypress test would still pass since it actually asserts a behavior of the application rather than an implementation. I'd remove revert this file. What do you think?

Reply:
Let's remove it.